src/Cooklang.l

```diff
@@ -10,8 +10,7 @@
 NEW_LINE            \x0a|\x0d|\xc2\x85|\xe2\x80\xa8|\xe2\x80\xa9
 WS_1                \x20|\xc2\xa0|\xe1\x9a\x80|\xe2\x80\x80|\xe2\x80\x81|\xe2\x80\x82|\xe2\x80\x83|\xe2\x80\x84|\xe2\x80\x85|\xe2\x80\x86|\xe2\x80\x87|\xe2\x80\x88|\xe2\x80\x89|\xe2\x80\x8a|\xe2\x80\xaf|\xe2\x81\x9f|\xe3\x80\x80|\x09
 WHITE_SPACE         {WS_1}+
-COOKLANG_CHAR       ">"|"~"|"@"|"#"|":"|"{"|"}"|"%"
-
+NOT_COOKLANG_CHAR   [^\[\-~@#{}% \t\n\r]
```

Comment:
I'm wondering why `-` is added here?

And why there's no `>` and `:`?

Reply:
hum... this is indeed confusing. I will add a comment and maybe rename the variable.

The idea is that `NOT_COOKLANG_CHAR` is used only in the `WORD` definition:
```
WORD                  {WHITE_SPACE}*({NOT_COOKLANG_CHAR}|("["+[^-])|("-"[^-]))+
```

* in this definition, `-` are special, because we need to check that we don't have 2 `-` together (in this case it's a comment)
* same for `[`
* regarding `>` and `:`, both characters are specific to metadata, and there are no limitation in the word. also the new definition of metadata doesn't require the word definition (the regex definition is : `">>"[^:]+":".+`) so we don't need to specify them here